code/defines/procs/admin.dm

```diff
@@ -39,14 +40,20 @@
 
 	. = ""
 
+	if(check_watchlist(key) && is_admin(src))
+		watchlisted = TRUE
+
 	if(key)
 		if(C && C.holder && C.holder.fakekey && !include_name)
 			if(include_link)
 				. += "<a href='?priv_msg=[C.findStealthKey()];type=[type]'>"
 			. += "Administrator"
 		else
 			if(include_link && C)
-				. += "<a href='?priv_msg=[C.UID()];type=[type]'>"
+				if(watchlisted)
+					. += "<a style='color: #db9737;' href='?priv_msg=[C.UID()];type=[type]'>"
```

Comment:
Not sure if its good practice to use color codes explicitly instead of a class.

Reply:
I agree with Kyet. Best to make a special class or reuse one. This will make it more robust when people want to reuse this later or when say darkmode support will be added

Reply:
It already supports darkmode, orange is fine in light and dark. I don't want to create a class just to make this 1 thing orange.